feature-libs/pickup-in-store/styles/_my-preferred-store.scss

```diff
@@ -1,6 +1,4 @@
 %cx-my-preferred-store {
-  @import 'bootstrap/scss/buttons';
```

Comment:
If it does not cause problems I think we should leave this in case of breaking changes. Since, an import cant be toggled with a feature flag let's leave a comment asking to remove it once it's made permanent.